### PR TITLE
Added goonlinetools.com

### PR DESCRIPTION
**Pull request by rahulkumarsingh73690**

### CODE REVIEW

**Comment by huguestennier** [comment]
Can you resize the thumbnail to the right 596x396 size please? https://github.com/jamstack/jamstack.org#contribute-a-site-example

**Comment by rahulkumarsingh73690** [comment]
> Can you resize the thumbnail to the right 596x396 size please? https://github.com/jamstack/jamstack.org#contribute-a-site-example

Image resized as per your request

**Comment by rahulkumarsingh73690** [comment]
Please review this PR  @tabar @huguestennier

File: src/site/_data/examples/goonlinetools.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title: GoOnlineTools
+description: 100% Free Online Tools Collection.
+link: 'https://goonlinetools.com'
+thumbnailurl: /img/examples/t2-media.png
```
Comment (zachleat) [inline src/site/_data/examples/goonlinetools.yml:4]: I think this is wrong, right? Needs to match the other file’s filename

**Review by zachleat — CHANGES_REQUESTED** [review]
I think the extension on the yaml file needs to be `.yaml` too.

File: src/site/_data/examples/goonlinetools.yml (hunk 1 above)
Comment (rahulkumarsingh73690) [reply src/site/_data/examples/goonlinetools.yml]: Fixed

**Comment by huguestennier** [comment]
Unless i'm mistaken, the website doesn’t look like the screen shot. Can you update it?

**Comment by rahulkumarsingh73690** [comment]
@huguestennier Added latest image

**Comment by huguestennier** [comment]
🎉

**Pull request merged**